hack/verify-flags-underscore.py

```diff
@@ -184,88 +116,26 @@ def get_flags(rootdir, files):
                 new_excluded_flags.add(flag)
             if not "-" in flag:
                 continue
-            if flag not in flags:
-                new_flags.add(flag)
+            flags.add(flag)
```

Comment:
we collect this set of flags to return, but we don't actually use the return value anymore. I don't think collecting this set is even necessary now.